app/assets/javascripts/trackset.js

```diff
@@ -272,7 +272,7 @@ TrackSet.prototype.UpdateRhombPattern = function(pattern) {
 		// update the rhombus version of the pattern
 
 		var trkId = this.rtracks[pattern.trackIndex];
-    var rpattern = rhomb._song._tracks.getObjById(trkId)._playlist[pattern.playlistId];
+    var rpattern = rhomb.getSong().getTracks().getObjById(trkId)._playlist[pattern.playlistId];

```

Comment:
This should be a call to getObjBySlot(), since trkId is actually an index and not a Rhombus object ID.